settings.xml

```diff
@@ -95,7 +95,7 @@
           </releases>
           <snapshots>
             <enabled>true</enabled>
-            <updatePolicy>never</updatePolicy>
+            <updatePolicy>always</updatePolicy>
```

Comment:
Maybe we just care about the update policy of dependencies. Did you find plugins with snapshot versions?